examples/plot_metrics_with_error.py

```diff
@@ -0,0 +1,176 @@
+
+
+def compute_error_metric(metric_value, sample_size, z_score):
+    """ Standard Error Calculation (Binary Classification)
+
+    Assumes infinitely large population,
+    Should be used when the sampling fraction is small.
+    For sampling fraction > 5%, may want to use finite population correction
+    https://en.wikipedia.org/wiki/Margin_of_error
+
+    Note: 
+        Returns absolute error (%)
+    """
+    return z_score*np.sqrt(metric_value*(1.0-metric_value))/np.sqrt(sample_size)
+
+
+def recall_wilson(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    tn, fp, fn, tp = confusion_matrix(y_true, y_pred).ravel()
+    bounds = wilson(tp/(tp+fn), tp + fn, digits_of_precision, z_score)
+    return bounds
+
+def recall_normal_err(y_t, y_p):
+    assert len(y_t) == len(y_p)
+    tn, fp, fn, tp = confusion_matrix(y_t, y_p).ravel()
+    error = compute_error_metric(tp/(tp+fn), tp + fn, z_score=z_score)
+    return (error, error)
+
+def accuracy_wilson(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    score = accuracy_score(y_true, y_pred)
+    bounds = wilson(score, len(y_true), digits_of_precision, z_score)
+    return bounds
+
+def accuracy_normal_err(y_true, y_pred):
+    assert len(y_true) == len(y_pred)
+    score = accuracy_score(y_true, y_pred)
+    error = compute_error_metric(score, len(y_true), z_score)
+    return (error, error)
+
+
+# %%
+# Classifier
+# ----------
+# Now we use a Decision Tree to make predictions
+#
+
+classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
+classifier.fit(X, y_true)
+y_pred = classifier.predict(X)
+
+# %%
+# MetricFrame
+# -----------
+# Now we create a MetricFrame to generate the Recall wilson bounds and Recall normal error
+#
+
+# Analyze metrics using MetricFrame
+metrics_dict = {
+    'Recall': recall_score,
+    'Recall Bounds': recall_wilson,
+    'Recall Error': recall_normal_err,
+    'Accuracy': accuracy_score,
+    'Accuracy Bounds': accuracy_wilson,
+    'Accuracy Error': accuracy_normal_err,
+}
+metric_frame = MetricFrame(metrics_dict, y_true, y_pred, sensitive_features=sex)
+
+# %%
+# plot metrics with (symmetric) error bars
+axs = plot_metric_frame(metric_frame, plot_type="scatter", 
+    metrics=['Recall', 'Accuracy'], 
+    error_bars=['Recall Error', 'Accuracy Error'])
+plot_metric_frame(metric_frame, plot_type="bar", metrics='Recall', error_bars='Recall Error')
+
+# %%
+# plot metrics with confidence intervals (possibly asymmetric)
+plot_metric_frame(metric_frame, plot_type="bar", metrics=['Recall', 'Accuracy'], conf_intervals=['Recall Bounds', 'Accuracy Bounds'])
+plot_metric_frame(metric_frame, plot_type="scatter", metrics='Recall', conf_intervals='Recall Bounds')
+
+# %%
+# plot metrics without error bars
+plot_metric_frame(metric_frame, plot_type="scatter", metrics=['Recall', 'Accuracy'])
+
+# %%
+# plots all columns and treats them as metrics without error bars
+axs = plot_metric_frame(metric_frame, plot_type="bar")
+
+# %%
+# Custom Plot 
```

Comment:
You will eventually need to expand these descriptions a bit, but let's agree on the code first.